newrelic/hooks/mlmodel_openai.py

```diff
@@ -209,28 +286,29 @@ def wrap_chat_completion_create(wrapped, instance, args, kwargs):
         "response.headers.ratelimitRemainingRequests": check_rate_limit_header(
             response_headers, "x-ratelimit-remaining-requests", True
         ),
-        "vendor": "openAI",
-        "ingest_source": "Python",
         "response.number_of_messages": len(messages) + len(choices),
     }
 
-    transaction.record_custom_event("LlmChatCompletionSummary", chat_completion_summary_dict)
+    transaction.record_custom_event("LlmChatCompletionSummary", full_chat_completion_summary_dict)
 
     message_list = list(messages)
+    choices_message = None
     if choices:
-        message_list.extend([choices[0].message])
+        choices_message = choices[0].message
```

Comment:
I think it's easier at this point to keep the choices list and the messages list separtate as that way we can tell which is sent vs received.